test/core.jl

```diff
@@ -28,45 +28,45 @@ let
     end
     let
         buffer = IOBuffer()
-        tape = Tape(1)
-        tape[1] = 5
-        show(buffer, tape)
+        tape_ = Tape(1)
```

Comment:
This is actually technically unnecessary since it's inside of a `let`, but whatevs